provide/Services/MessageProtocolParser.cs

```diff
@@ -0,0 +1,89 @@
+using System.IO;
+using System.Text;
+
+public class Message
+{
+    public string OpCode { get => "BLINE"; } // up to 40 bits
```

Comment:
opcode and type i added as readonly to simplify

Reply:
cool